.husky/pre-push

```diff
@@ -3,3 +3,4 @@
 
 eslint --max-warnings 0 src --ext .ts --ext .tsx --cache
 npm test  -- --watchAll=false
+git add -A src
```

Comment:
I dont think this is a good idea. Maybe just add the folder with the snapshots?

Reply:
y i dont know it isnt that important